tvb_contrib/tvb/contrib/cosimulation/cosim_monitors.py

```diff
@@ -97,15 +97,19 @@ class CosimMonitorFromCoupling(CosimMonitor):
                dynamic equations of the Model. Its primary purpose is to 'rescale' the
                incoming activity to a level appropriate to Model.""")
 
-    synchronization_n_step = None
+    min_idelay = Int(
```

Comment:
You should not define it as a parameter because the value defined by the user won't be used. 
min_idelay should be defined as a private variable, i.e. _min_idelay without defining the type.

Reply:
Sorry for not noticing it! I had to also correct in the CoSimulator, and this is where I got confused and forgot it!
Done now!
Tests still PASS.